public/js/scene.js

```diff
@@ -33,6 +33,8 @@ export class Scene {
     }
 
     openReading() {
+        this.game.style.display = "flex";
```

Comment:
Do this in the CSS file, not from JS.

Reply:
this is just a temporary fix, just so it looks presentable for the presentation